pex/resolver.py

```diff
@@ -610,6 +611,90 @@ def build_wheels(
         return build_results
 
 
+@attr.s(frozen=True)
+class DirectRequirements(object):
+    @classmethod
+    def calculate(
```

Comment:
Is the contents of this method just moved, with no changes?

Reply:
Yup.